content/pt-br/docs/reference/glossary/persistent-volume.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Volume Persistente
+id: persistent-volume
+date: 2018-04-12
+full_link: /pt-br/docs/concepts/storage/persistent-volumes/
+short_description: >
+  Um objeto de API que representa uma parte do armazenamento no cluster. Disponível como um recurso conectável geral que persiste além do ciclo de vida de qualquer pod individual.
```

Comment:
"recurso geral e conectável" instead of "recurso conectável geral". See also the line 14